integration-test/2022-disputed-borders.py

```diff
@@ -126,3 +126,34 @@ def test_admin_level_3_other_place(self):
             z, x, y, 'boundaries', {
                 'id': 345678,
             })
+
+    def test_disputed_by_to_unrecognized_disputed_reference_line(self):
+        z, x, y = (16, 53533, 28559)
+
+        self.generate_fixtures(
+            # https://www.openstreetmap.org/relation/13574166
+            dsl.way(13574166, dsl.tile_diagonal(z, x, y), {
+                'admin_level': '2',
+                'boundary': 'disputed',
+                'disputed_by': 'RU;PK;IL;PS;SA;EG;ID;BD',
+                'name': '1949 Israeli-Syrian DMZ',
+                'ne:brk_a3': 'B16',
+                'ne_id': '1746705859;1746705871',
```

Comment:
Should this be `ne:id`? If it's not that in OSM.org then the change might need to be 2 places, not just here.

Reply:
I cut and pasted this from OSM, though, so it is def `ne_id`. https://www.openstreetmap.org/relation/13574166

Reply:
@jeffdefacto is there a reason you went with `ne_id` not `ne:id` when adding these to OSM?

Reply:
Thats how they were tagged in the shapefile by Stephen I guess, when I received it and I just went with it. I can update the features to change it though.

Reply:
Let's leave it as-is. It is "ne_id" in Natural Earth native, so it's copied over to OSM that way, and that's how the data is represented in OSM now. It is slightly odd to see mix of `ne:*` and `ne_id`, but it's fine.